README.md

```diff
@@ -958,6 +963,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**Luanti**](https://github.com/minetest/minetest) <sup>**[[F-Droid](https://f-droid.org/packages/net.minetest.minetest)]**</sup>
 * [**Pocket Broomball**](https://github.com/dulvui/pocket-broomball) <sup>**[[F-Droid](https://f-droid.org/packages/com.simondalvai.pocketbroomball)]**</sup>
 * [**Quinb**](https://gitlab.com/deepdaikon/Quinb) <sup>**[[F-Droid](https://f-droid.org/packages/xyz.deepdaikon.quinb)]**</sup>
+* [**Space Vertex Multiplayer**](https://github.com/smart-fun/spacevertex) <sup>**[[F-Droid](https://f-droid.org/packages/fr.arnaudguyon.spacevertex)]**</sup>
```

Comment:
Same as mentioned above; new but on F-Droid.

Reply:
Tested on personal device, works fine. There are very few multiplayer games at F-Droid. I think good to have _local multiplayer_ games on the list, even if they are new.